pkg/platform/model/buildplanner/impactreport.go

```diff
@@ -0,0 +1,33 @@
+package buildplanner
+
+import (
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/request"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/response"
+)
+
+type ImpactReportParams struct {
+	Owner      string
+	Project    string
+	BeforeExpr []byte
+	AfterExpr  []byte
```

Comment:
We cannot use commit IDs because at this time they must be publicly attached to the project. Local commit IDs will not work. I've filed PB-5177 for this.